pkg/controller/kubelet-config/kubelet_config_features.go

```diff
@@ -203,17 +203,6 @@ func generateKubeConfigIgnFromFeatures(cc *mcfgv1.ControllerConfig, templatesDir
 		updateOriginalKubeConfigwithNodeConfig(nodeConfig, originalKubeConfig)
 	}
 
-	defaultFeatures, err := generateFeatureMap(createNewDefaultFeatureGateAccess(), openshiftOnlyFeatureGates...)
```

Comment:
This logic also exists in the kubeletconfig generation, so we should clean this up (but not as part of this PR)

We should also look to remove this whole sub-controller if that's possible with the new workflow